kubernetes/.template.values.yaml

```diff
@@ -1,13 +1,15 @@
 global:
   frontDoor: GAME_FRONT_DOOR
+  frontDoorAlt: GAMEON_DUMMY
   frontDoorHost: GAMEON_INGRESS
+  frontDoorAltHost: GAMEON_DUMMY_INGRESS
   mode: development
   data:
 # Global: interservice communication
     MAP_SERVICE_URL: http://map.gameon-system.svc.cluster.local:9080/map/v1/sites
     MAP_HEALTH_SERVICE_URL: http://map.gameon-system.svc.cluster.local:9080/map/v1/health
     PLAYER_SERVICE_URL: http://player.gameon-system.svc.cluster.local:9080/players/v1/accounts
-    RECROOM_SERVICE_URL: wss://GAME_FRONT_DOOR/rooms
+    RECROOM_SERVICE_URL: ws://room.gameon-system.svc.cluster.local:9080/rooms
```

Comment:
We wanted to treat room as a proper 3rd party service (and it can be tested using primitive UI) .. we routed it through the front door deliberately... do we really want to drop this?